community/resources/resources.md

```diff
@@ -5,21 +5,11 @@ redirect_from:
   - /community/resources/index.html
 ---
 
-This Community collection of resources links to blog posts, podcasts, presentations, and books created by members of the [Magento](https://glossary.magento.com/magento) community and core developers separated into popular topics.
+## Magento blog
 
-New items will be added incrementally, and older links rotated out. Keep an eye on this space for the latest community resources!
+The [Magento blog][0] is a great resource for news about Magento, new community resources (articles, tutorials, etc.), events and other Magento-related items.
```

Comment:
```suggestion
The [Magento DevBlog][0] is a great resource for news about Magento, new third-party community resources (articles, tutorials, etc.), events, and other Magento-related items.
```

Reply:
Shouldn't we be linking to the DevBlog instead of the Marketing blog? Perhaps we should link to both, but I know that Sherrie posts links to third-party resources on the DevBlog.